docs/update-gh-pages.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# This script updates the documentation on knora.org by pushing a commit to the gh-pages branch
+# of the Knora repository on GitHub.
+
+# Before you run this script, you must be on the develop branch, having committed any changes
+# you made there.
+
+# If you don't want git to ask you for your username and password, use SSH instad of HTTPS
+# to clone the Knora repository.
+
+TMP_HTML="/tmp/knora-html"
+
+set -e
+
+# Build the HTML docs.
+
```

Comment:
add a check that the current branch is `develop`:
```
branch=`git rev-parse --abbrev-ref HEAD`
``` 
http://stackoverflow.com/questions/1593051/how-to-programmatically-determine-the-current-checked-out-git-branch